#include<iostream>
using namespace std;

int get()
{
	int num;
	cin>>num;
	if(num < 0 || num > 9)
		throw "input fault 예외 발생";
	return num;
}

int main()
{
	int num1, num2;
	while(1)
	{
		try{
			cout<<"0~9사이의 정수 입력 >> ";
			num1 = get();
			cout<<"0~9사이의 정수 입력 >> ";
			num2 = get();
			cout<<num1<<" X "<<num2<<" = "<<num1*num2<<endl;
		}
		catch(char *s){
			cout<<s<<"계산할 수 없음"<<endl;
		}
	}
}